app/containers/RightMenu/index.js

```diff
@@ -65,7 +66,11 @@ const RightMenu = (props, context) => {
       <div style={{ marginBottom: 64 }}>
         <Divider />
         <Grid container direction="row">
-          <Grid item xs={6}><IconButton className={classes.button}><Settings /></IconButton></Grid>
+          <Grid item xs={6}>
+            <IconButton className={classes.button} onClick={() => context.router.push(profilePage())}>
```

Comment:
1. `context` to jest coś czego nie używamy z wyjątkiem ekstremalnych sytuacji
2. nie chcemy tego robić po przez `onClick` bo wtedy nie można otworzyć w nowej karcie tyko przez `Link`